release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -68,6 +68,21 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 [id="ocp-4-12-ibm-power"]
 === IBM Power
 
+[id="ocp-4-12-images"]
+=== Images
+
+A new import value, `importMode`, has been added to the to the `importPolicy` parameter of image streams. The following fields are available for this value: 
+
+* `Legacy`: `Legacy` is the default value for `importMode`. When active, the manifest list is discarded, and a single sub-manifest is imported. The platform is chosen in the following order of priority: 
++
+. Tag annotations
+. Control plane architecture
+. Linux/AMD64
+. The first manifest in the list 
+
+* `PreserveOriginal`: When active, the original manifest is preserved. For manifest lists, the manifest list and all of its sub-manifests are imported. 
+
```

Comment:
When i last checked on this - if importMode is changed, the tag needs to also be changed for an import to happen. I believe this would need to be made so that imports happen automatically with the existing tag on a change. is that still the case @flavianmissi ?

Reply:
We have a fix for it @Prashanth684. It's not in yet (we're waiting on https://github.com/openshift/openshift-apiserver/pull/320), but we're hoping to land it next week.

Reply:
Does https://github.com/openshift/openshift-apiserver/pull/320 impact this release note @flavianmissi ?

Reply:
Nope, that was just a blocker @stevsmit. The PR fixing the behaviour Prashanth is describing is https://github.com/openshift/openshift-apiserver/pull/325 - we're aiming to get it merged before branching day but if we don't manage to then I guess we'd need to document it somehow.